frontend/pages/game_page.css

```diff
@@ -0,0 +1,42 @@
+.red { color: red; }
+.green { color: green; }
+.blue { color: blue; }
+.magenta { color: magenta; }
+.purple { color: purple; }
+.orange { color: orange; }
+body { 
+    text-align: center; 
+    background-image: url(https://wallpapercave.com/wp/wp2675347.jpg);
+    background-position: center;
+    margin: 0;
+    height: 90vh;
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+h1 { 
```

Comment:
Could we use CSS classes instead of global tag classes for these? e.g `.tetris-header` instead of `h1`